pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1321,6 +1321,12 @@ type ObjectStoreSpec struct {
 	// +optional
 	// +nullable
 	Security *SecuritySpec `json:"security,omitempty"`
+
+	// Network related configuration
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// +nullable
+	// +optional
+	Network NetworkSpec `json:"network,omitempty"`
```

Comment:
Let's make this nullable so if it is not specified, we will fall back to the network settings on the cluster CR
```suggestion
	Network *NetworkSpec `json:"network,omitempty"`
```

Reply:
done